### PR TITLE
Hacking/strict field validation

### PR DESCRIPTION
**Pull request by babubabu**
Zuerst hab ich mal die Grundstruktur + die Anpassung im ContactPersonDto gemacht. Wenn das für euch passt, dann ziehe ich das auch auf die anderen DTos um

### CODE REVIEW

File: backend/src/main/java/quarano/core/MessageConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import org.springframework.context.support.ReloadableResourceBundleMessageSource;
+import org.springframework.validation.Validator;
+import org.springframework.validation.beanvalidation.LocalValidatorFactoryBean;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+/**
+ * @author Felix Schultze
+ */
+@Configuration
+public class MessageConfiguration implements WebMvcConfigurer {
```
Comment (odrotbohm) [inline backend/src/main/java/quarano/core/MessageConfiguration.java:16]: Why do we need this explicit configuration? Everything but the `MessageSourceAccessor` that was previously already declared in `Quarano` is exactly what Boot configures by default.

File: backend/src/main/java/quarano/core/validation/AlphaNumeric.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ */
+@Pattern(regexp = Strings.LETTERS, message = ERROR_MSG)
+@Documented
+@Target({ METHOD, FIELD, ANNOTATION_TYPE, CONSTRUCTOR, PARAMETER, TYPE_USE })
+@Retention(RUNTIME)
+@Constraint(validatedBy = {})
+public @interface AlphaNumeric {
+    String ERROR_MSG = "{Pattern.alphaNumeric}";
+
+    String message() default ERROR_MSG;
```
Comment (odrotbohm) [inline backend/src/main/java/quarano/core/validation/AlphaNumeric.java:46]: Do we need these fields if we don't make any use of them anywhere for now?

Comment (babubabu) [reply backend/src/main/java/quarano/core/validation/AlphaNumeric.java:46]: Unfortunately, they are needed at @Constraint annotations. If they are not provided, we'll get a RuntimeException on Request for each of these 3 fields.
HV000074: quarano.core.validation.Name contains Constraint annotation, but does not contain a groups parameter.

File: backend/src/main/java/quarano/core/MessageConfiguration.java (hunk 1 above)
Comment (babubabu) [reply backend/src/main/java/quarano/core/MessageConfiguration.java:16]: Its strange. We need to return the Validator to resolve the error messages. If I just return an autowired LocalValidatorFactoryBean, it's not working. I have to set the messagesource on the LocalValidatorFactoryBean manually. So I decieded to define it completely

Comment (odrotbohm) [reply backend/src/main/java/quarano/core/MessageConfiguration.java:20]: JSR 303 validation has already worked before in general, that's why I was puzzled.

**Pull request closed without merging**